### PR TITLE
Add TimerController unit tests

### PR DESCRIPTION
**Pull request by fjvallarino**
@Rise-Vision/apps please review

### CODE REVIEW

File: src-js/Viewer/Content/TimerController.tests.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import {stub, spy, useFakeTimers} from 'sinon';
+
+require('../../');
+
+var RiseVision = window.RiseVision;
+
+describe('GadgetController: ', function() {
```
Comment (alex-deaconu) [inline src-js/Viewer/Content/TimerController.tests.js]: @fjvallarino wondering if you can update the controller name here. Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino just one typo indicated in the comments. Otherwise, looks good 👍 . Thanks!

**Comment by fjvallarino** [comment]
@Rise-Vision/apps updated

**Comment by alex-deaconu** [comment]
@fjvallarino thanks for fixing 👍

**Pull request merged**